api/api/admin/__init__.py

```diff
@@ -83,3 +86,54 @@ class ProviderAdmin(admin.ModelAdmin):
     list_display = ("provider_name", "provider_identifier", "media_type")
     search_fields = ("provider_name", "provider_identifier")
     ordering = ("media_type", "provider_name")
+
+
+@admin.register(ThrottledApplication)
+class ThrottledApplicationAdmin(admin.ModelAdmin):
+    search_fields = ("client_id", "name")
+    list_display = ("client_id", "name", "user", "rate_limit_model")
+    ordering = ("client_id",)
```

Comment:
The `user` field isn't very useful here, but the date the app was created is! It will also be nice to sort it to show first the most recently created apps. The client_id is quite random. Here is my suggestion of how the code should look:
```suggestion
    list_display = ("client_id", "name", "created", "rate_limit_model")
    ordering = ("-created",)
```